Tasks/AzureRmWebAppDeployment/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -165,7 +165,7 @@
   "loc.messages.ScriptStatusTimeout": "Unable to fetch script status due to timeout.",
   "loc.messages.PollingForFileTimeOut": "Unable to fetch script status due to timeout. You can increase the timeout limit by setting 'appservicedeploy.retrytimeout' variable to number of minutes required.",
   "loc.messages.InvalidPollOption": "Invalid polling option provided: %s.",
-  "loc.messages.MissingAppTypeWebConfigParameters": "Attribute '-appType' is missing in the Web.config parameters. Valid values for '-appType' are: 'python_Bottle', 'python_Django', 'python_Flask' and 'node'.<br />For example, '-appType python_Bottle' (sans-quotes) in case of Python Bottle framework..",
+  "loc.messages.MissingAppTypeWebConfigParameters": "Attribute '-appType' is missing in the Web.config parameters. Valid values for '-appType' are: 'python_Bottle', 'python_Django', 'python_Flask', 'node' and 'Go'.<br />For example, '-appType python_Bottle' (sans-quotes) in case of Python Bottle framework..",
```

Comment:
I assume this is not coming as part of your change.

Reply:
yes. not from my current change.